proxy/proxy.go

```diff
@@ -105,6 +105,11 @@ func (s *Handler) doHijack(message *common.HttpMessage, key string, incomingMess
 			log.WithField("error", err).Errorf("Failed to read request %s", u.Host)
 		}
 		reader.Close()
+
+		for range incomingMessages {
+			// waiting for channel to close
+		}
+		conn.Close()
```

Comment:
I guess the "right" way to end the transmission would be to to send a websocket close frame, but we aren't really setup for that here.